### ISSUE TITLE
500 error - 

### ISSUE THREAD

**Issue by BritishBenji** [SuperSonicHub1/TikTok-RSS#4]
Sorry for only ever opening issues here, and not bringing good news, but:
```
[2022-01-26 14:12:10,940] ERROR in app: Exception on /feed/LindseyStirling [GET]
Traceback (most recent call last):
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\models.py", line 910, in json
    return complexjson.loads(self.text, **kwargs)
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\json\__init__.py", line 346, in loads
    return _default_decoder.decode(s)
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\json\decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 2073, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1518, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1516, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\flask\app.py", line 1502, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**req.view_args)
  File "C:\Users\25030\Desktop\TikTok-RSS\main.py", line 16, in feed
    feed = generate_feed(username)
  File "C:\Users\25030\Desktop\TikTok-RSS\generator.py", line 56, in generate_feed
    body = res.json()
  File "C:\Users\25030\AppData\Local\Programs\Python\Python310\lib\site-packages\requests\models.py", line 917, in json
    raise RequestsJSONDecodeError(e.msg, e.doc, e.pos)
requests.exceptions.JSONDecodeError: [Errno Expecting value] : 0
127.0.0.1 - - [26/Jan/2022 14:12:10] "GET /feed/LindseyStirling HTTP/1.1" 500 -
```

I have some free time, so I'll have a look and see if I can find what's causing the issue myself, but putting this here in case anyone is better at this than I am :)

**Comment by BritishBenji** [comment]
`client.call_api` seems to return a 200 response but contains no content

**Comment by SuperSonicHub1** [comment]
I'll check it out later, probably an authentication issue.

**Comment by SuperSonicHub1** [comment]
This seems to be an issue `yt-dlp` is also having. We'll need to wait on them:
* https://github.com/yt-dlp/yt-dlp/issues/2534
* https://github.com/yt-dlp/yt-dlp/issues?q=is%3Aopen+tiktok

**Comment by SuperSonicHub1** [comment]
Come back to me in about a week and I'll check on this again.

**Comment by BritishBenji** [comment]
Issue fixed in commit #2161 https://github.com/yt-dlp/yt-dlp/pull/2161
